app/models/manageiq/providers/kubernetes/container_manager/metrics_capture/hawkular_client.rb

```diff
@@ -7,4 +7,8 @@ def initialize(ext_management_system)
 
     hawkular_client
   end
+
+  def system_metrics
+    @client.http_get('/metrics')
```

Comment:
@simon3z yes it can be large when the system has lots of nodes,
but I do not have a better way of testing the authentication :partly_sunny: 

a. I will look for a better way here ( do you have a direction to look at ? ):
http://www.hawkular.org/docs/rest/rest-metrics.html#_availability 

b. here is an example of this call on the qa system [L615](https://github.com/yaacov/manageiq/blob/bebcf7e2c8abddb145ae50b80e28481c17655c22/spec/vcr_cassettes/manageiq/providers/kubernetes/container_manager/metrics_capture/capture_context_system_metrics.yml#L615)